Dockerfile

```diff
@@ -1,20 +1,34 @@
-FROM python:3.6.4
-RUN apt-get update && apt-get -y install \
-    gnupg2 \
-    pass \
+ARG PYTHON_VERSION=3.7
+
+FROM python:${PYTHON_VERSION}-alpine
+RUN apk add --no-cache \
+    coreutils \
+    gnupg \
     curl
 
+# TODO remove once pass is in the main repository
+RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing pass
```

Comment:
I should probably note here that edge packages are not guaranteed to work with stable, so while this one currently does, it's possible that this may break on a future update of stable (or the edge package)